src/groups/groupForm.jsx

```diff
@@ -0,0 +1,85 @@
+import { hot } from 'react-hot-loader';
+import React from 'react';
+import {
+    Typography,
+    Divider,
+    Form,
+    Input,
+    Button,    
+    Select,
+    Breadcrumb,
+  } from 'antd';
+
+export default class GroupForm extends React.Component {
+    constructor(props) {
+      super(props)
+      this.clearForm = this.clearForm.bind(this)
+    }
+
+    clearForm(){
+        alert('Limpiar formulario')
+    }
+    state = {
+        memberItems: ['Marcos','Louis','Felipe'],
+    }
+    render() {
+        const {memberItems} = this.state
+        const { Title } = Typography;
+        const { Option } = Select
+ 
+       
+        const layout = {
+            labelCol: { span: 7 },
+            wrapperCol: { span: 12 },
+        }
+        const tailLayout = {
+          wrapperCol: { offset: 7, span: 12 },
+        }
+        return(            
+            <>
+            <Breadcrumb>
+                <Breadcrumb.Item>Grupos</Breadcrumb.Item>
+                <Breadcrumb.Item>Crear</Breadcrumb.Item>
+            </Breadcrumb>
+            <div className="project-form-container" style={{margin: "50px"}}>
+                <Title level={3}>Crear grupo</Title>
+                <Divider dashed></Divider>
+                <Form {...layout}
+                    name="projectForm"
+                    layout="horizontal"
+                >
+                    <Form.Item 
+                        label="Nombre"
+                        name="projectName"
```

Comment:
the "name" attribute is useful for submitting data. Here we are talking about Groups so I think we should change to "groupName"